analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.CSharp8.cs

```diff
@@ -38,7 +47,7 @@ void DoDispose()
     {
         var d = new Disposable();
         d.Dispose();
-        d.Dispose(); // FIXME Non-compliant
+        d.Dispose(); // Noncompliant
```

Comment:
Asserted to `Resource 'd' has already been disposed explicitly or implicitly through a using statement. Please remove the redundant disposal.`

Reply:
I still read it like this `Resource 'd' has already been disposed (explicitly or implicitly) through a using statement.` instead of `Resource 'd' has already been disposed explicitly or (implicitly through a using statement).`.

To improve later

Reply:
I did a small improvement - but nothing great.

Reply:
It's clear now.